Formula/salt.rb

```diff
@@ -276,6 +271,11 @@ def caveats
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/salt --version")
+    output = shell_output("#{bin}/salt-call --local --config-dir=#{testpath} --log-file=/dev/null test.versions")
```

Comment:
```suggestion
    output = shell_output("#{bin}/salt --config-dir=#{testpath} --log-file=/dev/null --versions")
```
From https://github.com/cdalvaro/homebrew-tap/pull/37/commits/b9b7d591d09d7d55985e67c55559daf4813b10b0